src/lib/components/blog/post-meta.svelte

```diff
@@ -3,7 +3,9 @@
     import { handleCopy } from '$lib/utils/copy';
     import { formatDate } from '$lib/utils/date';
     import type { AuthorData } from '$routes/blog/content';
-    import { page } from '$app/state';
```

Comment:
Removed page component as this was not being used in this file ! 
Simple as that !